### DISCUSSION TITLE
is this maintained?

### DISCUSSION THREAD

**Discussion by midas0441** [arnm/ob-mermaid#28] [General]
Is this a stalled/suspended project, or is the plan to actively develop it? I found `mermaid-mode` which is certainly different (not org-babel).

**Comment by arnm** [comment]
@midas0441 

Hey there, I realize this is a bit of a late response but now that I got time to devote to open source I want to keep the discussion going. 

I think the plan now is to see what people have suggested already and go from there. Definitely willing to discuss new ideas and work with the community on this together 🤝
